projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib-checkout.migration.ts

```diff
@@ -768,7 +759,7 @@ export const CART_LIB_AND_CHECKOUT_MIGRATION: ConstructorDeprecation[] = [
       },
       {
         className: 'CheckoutLegacyStoreEventListener',
-        importPath: '',
+        importPath: '@spartacus/checkout/base/root',
```

Comment:
TODO: I think CheckoutLegacyStoreEventListener is not exported in the public API and might be missing in an index file.

Reply:
Created #15310 to follow-up